src/app/system/manage-data-tables/view-data-table/view-data-table.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+
+    <button mat-raised-button color="primary" [routerLink]="['../../edit', dataTableData.registeredTableName]" >
+        <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+        Edit
+    </button>
+
+    <button mat-raised-button color="warn" (click)="delete()">
+        <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+        Delete
+    </button>
+
+</div>
+
+<div class="container m-b-15" fxLayoutAlign="start">
+    <mat-card>
+        <mat-card-subtitle>Associated With {{ dataTableData.applicationTableName }}</mat-card-subtitle>
```

Comment:
Don't add as a subtitle as it seems like disabled content. Add a class setting the font-weight property for `Associated With` keeping the rest of the content normal. See `accounting section` for reference to add the class and make sure the name of class is meaningful.